src/test/server/store-test.js

```diff
@@ -0,0 +1,118 @@
+const UserAction = require('../../../_compiled_/actions/user-action').default;
+const UserInfo   = require('../../../_compiled_/stores/user-info').default;
+const assert     = require('assert');
+const usersData  = require('../../data/users.json');
+const UserStore  = require('../../../_compiled_/stores/user-store').default;
+
+let users;
```

Comment:
let's use const and call it e.g. `initialUsers`. We can init it before tests.
we should not change this object during testsing

Reply:
It's better to extract method e.g. `_initUsers` or `_initUserList` in the store and use it in the constructor 
```js
usersData.map((user) => {
            return new UserInfo(
                {
                    id:      user.id,
                    name:    user.name,
                    phone:   user.phone,
                    email:   user.email,
                    website: user.website
                },
                user.address,
                user.company
            );
        });
```
In tests we will be able to restore initial list after each test